game/plugin/cmd/src/teleport-cmd.plugin.kts

```diff
@@ -75,6 +75,30 @@ on_command("tele", PrivilegeLevel.ADMINISTRATOR)
         }
     }
 
+/**
+ * Teleports the player to another player.
+ */
+on_command("teleto", PrivilegeLevel.ADMINISTRATOR)
+    .then { player ->
+        val invalidSyntax = "Invalid syntax - ::teleto [player name]"
+
+        if (arguments.size == 1) {
+            val playerName = arguments[0]
+            val foundPlayer = player.world.getPlayer(playerName)
+
+            if(foundPlayer == null) {
+                player.sendMessage("Player $playerName is currently not online.")

```

Comment:
Maybe `"Player $playerName is currently offline or does not exist."`

Reply:
Addressed in latest commit.